dashboard/app/views/levels/_blockly.html.haml

```diff
@@ -6,6 +6,10 @@
 - level['showFinish'] = local[:embed]
 
 -if local[:embed]
+  :css
+    #blockly.readonly .userHidden {
```

Comment:
Rather than overloading `.readonly`, prefer setting `.embed` on `#blockly` and then adding the style to `/blockly-core/core/ui/css.js`.  That way any future embed-specific styles also get easier to write.